test/model/TaskManagerTest.cpp

```diff
@@ -299,6 +299,84 @@ TEST_F(TaskManagerTest, shouldAddMultipleLabels){
     EXPECT_EQ(label2, tm.getTasks()[0].data().labels()[1]);
 }
 
+TEST_F(TaskManagerTest, shouldAddMultipleLabelsClearOne){
+    TaskManager tm;
+    Core::Task::Priority p = Core::Task::Priority::Task_Priority_NONE;
+    Core::Task t;
+    t.set_title("Task");
+    t.set_priority(p);
+    t.set_due_date(time(nullptr));
+    t.set_is_complete(false);
+    Core::TaskID id = *tm.Add(t).id;
+    std::string label = "testing";
+    std::string label2 = "testing2";
+    tm.AddLabel(id, label);
+    tm.AddLabel(id, label2);
+    tm.ClearLabel(id, label);
+    auto tasks = tm.getTasks();
+    EXPECT_EQ(1, tasks[0].data().labels().size());
```

Comment:
better to use ASSERT_EQ when checking container size to fail test before out of range exception

Reply:
done